Extension/browser/chrome/lib/content-script/content-script.js

```diff
@@ -19,4 +19,38 @@ var contentPage = {
     onMessage: new OnMessageEvent()
 };
 
-var i18n = chrome.i18n;
\ No newline at end of file
+var i18n = chrome.i18n;
+
+//TODO: Move it somewhere
+function pageMessageListener(event) {
```

Comment:
Ah, got it. Ok, anyway, it should be in some separate file (or at least inside preload.js).